R/io.R

```diff
@@ -5,10 +5,13 @@
 #'
 #' Writes a full mzQC object to disk.
 #'
+#' You can in theory also provide any mzQC subelement,
+#' but the resulting mzQC file will not validate since its incomplete.
+#'
 #' The filename should have an '.mzQC' as suffix (warning otherwise).
 #'
-#' @param filepath A filename (with path) to write to.
-#' @param mzqc_obj An mzQC object, which is serialized to JSON and then written to disk
+#' @param filepath A filename (with optional path) to write to.
+#' @param mzqc_obj An MzQCmzQC root object, which is serialized to JSON and then written to disk
```

Comment:
So the object notation is a bit confusing. Do you have a quick overview in the doc's somewhere, that explains what's why what?

Reply:
the prefix `MzQC` is used for all datastructures related to the mzQC schema.
Im a bit unhappy with the naming as well.
Recommendations welcome.
I think a common prefix for these objects is a good thing when using an IDE.